spec/features/front_matter_reviewer_report_spec.rb

```diff
@@ -67,189 +67,192 @@ def create_reviewer_report_task
   before do
     assign_reviewer_role paper, reviewer
     FactoryGirl.create :feature_flag, name: "PREPRINT"
-
-    login_as(reviewer, scope: :user)
-    visit "/"
   end
 
   scenario "A paper's creator cannot access the Reviewer Report" do
     create_reviewer_invitation(paper)
     reviewer_report_task = create_reviewer_report_task
-    wait_for_ajax
     ensure_user_does_not_have_access_to_task(
       user: paper.creator,
       task: reviewer_report_task
     )
   end
 
-  scenario 'A reviewer can fill out their own Reviewer Report, submit it, and see a readonly view of their responses' do
-    create_reviewer_invitation(paper)
-    reviewer_report_task = create_reviewer_report_task
+  context 'reviewer is logged in' do
+    before do
+      login_as(reviewer, scope: :user)
+      visit "/"
+    end
```

Comment:
This is the only meaningful change in this file right? Everything else is just indentation noise?

Reply:
Yeah, sorry...